app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Far Mar | Alice &amp; Ashley</title>
+  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
+  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
+  <%= csrf_meta_tags %>
+</head>
+<body>
+
+<header>
+  <nav>
+    <%= render "layouts/#{@nav}_nav" %>
```

Comment:
Thanks for giving us feedback on this -- we went back and forth on several options for this and got outside input, and no one could agree on whether these should be all in one partial or separated, and the specific implementation of each of those options, considering the messy routing issues + needing different navs for the VendorsController#show action depending on the user type but relying on the route to determine user type since we haven't learned about user authentication yet.